rules/cloud/aws/aws_attached_malicious_lambda_layer.yml

```diff
@@ -11,8 +11,13 @@ logsource:
 detection:
     selection:
         eventSource: lambda.amazonaws.com
+    filter1:
         eventName: UpdateFunctionConfiguration
-    condition: selection
+    filter2:
+        eventName: UpdateFunctionConfiguration20150331
+    filter3:
+        eventName: UpdateFunctionConfiguration20150331v2
+    condition: selection and (filter1 or filter2 or filter3)
```

Comment:
The same detection but more compact
```yaml
    selection:
        eventSource: lambda.amazonaws.com
        eventName: 
            - UpdateFunctionConfiguration
            - UpdateFunctionConfiguration20150331
            - UpdateFunctionConfiguration20150331v2
    condition: selection
```
otherwise to get ready for next UpdateFunctionConfigurationyyyymmddvX
```yaml
    selection:
        eventSource: lambda.amazonaws.com
        eventName|startswith: UpdateFunctionConfiguration
    condition: selection
```

Reply:
Thanks @frack113